### PR TITLE
correct console output sentence

### PR DESCRIPTION
**Pull request by cre8**
correct the console output that doesn't make sense.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -207,4 +207,4 @@ var broadcast = (message) => sockets.forEach(socket => write(socket, message));
 
 connectToPeers(initialPeers);
 initHttpServer();
```
Comment (lhartikk) [inline main.js]: IMO this pull requess should not modify this line at all

File: main.js [hunk 2]
```diff
@@ -207,4 +207,4 @@ var broadcast = (message) => sockets.forEach(socket => write(socket, message));
...
-initP2PServer();
```
Comment (lhartikk) [inline main.js]: IMO this pull request should not modify this line at all

**Comment by lhartikk** [comment]
Thanks for the pull request!

I will merge this PR if you address the issue I commented in the "code" section

**Comment by cre8** [comment]
@lhartikk fixed it. I updated it directly via github and that also changed the end of file.

**Pull request merged**